lib/slices/createCitySlice.js

```diff
@@ -0,0 +1,34 @@
+import { countries } from "../db";
+
+import { v4 as uuidv4 } from "uuid";
+export const createCitySlice = (set) => ({
+  countries,
+  addCity: (FormData) => {
+    const { city, country, startDate, endDate, note, hotels, places, food} = FormData;
+    set((state) => ({
+      countries: state.countries.map((_country) => {
+        if (_country.id === country) {
```

Comment:
```suggestion
        if (country.id === countryID) {
```
As mentioned above, this could look like this if you make that little change above (and everywhere else, of course).